core/src/main/java/org/apache/iceberg/TableProperties.java

```diff
@@ -349,4 +350,27 @@ private TableProperties() {}
 
   public static final String UPSERT_ENABLED = "write.upsert.enabled";
   public static final boolean UPSERT_ENABLED_DEFAULT = false;
+
+  public static final String ENCRYPTION_TABLE_KEY = "encryption.table.key.id";
+
+  public static final String ENCRYPTION_DEK_LENGTH = "encryption.data.key.length";
+  public static final int ENCRYPTION_DEK_LENGTH_DEFAULT = 16;
+
+  public static final String ENCRYPTION_DATA_ALGORITHM = "encryption.data.algorithm";
+  public static final String ENCRYPTION_DATA_ALGORITHM_DEFAULT =
+      EncryptionAlgorithm.AES_GCM.toString();
+
+  /**
+   * Leverage file format native encryption instead of encrypting the entire file through Iceberg
+   * encryption stream
+   */
+  public static final String ENCRYPTION_PUSHDOWN_ENABLED = "encryption.pushdown";
+
+  public static final boolean ENCRYPTION_PUSHDOWN_ENABLED_DEFAULT = true;
```

Comment:
since it's the only option today (with false, we can only throw an exception), lets remove the "encryption.pushdown" parameter altogether. Less complexity / confusion, and one less config. When can add it later if/when needed.